app/services/catalog/add_to_order.rb

```diff
@@ -9,6 +9,7 @@ def initialize(params)
     def process
       order = Order.find_by!(:id => @params[:order_id])
       @order_item = order.order_items.create!(order_item_params.merge!(:service_plan_ref => service_plan_ref))
+      @order_item.update_message("info", "Order item tracking ID (x-rh-insights-request-id): #{@order_item.insights_request_id}")
```

Comment:
@syncrou Moving the logic here ends up being cleaner and a smaller PR change overall.